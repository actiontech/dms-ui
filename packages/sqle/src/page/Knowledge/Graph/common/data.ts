import { SigmaContainerProps } from '@react-sigma/core';
import { EdgeType, NodeType } from '../index.type';
import { Attributes } from 'graphology-types';

export const sigmaSettings: SigmaContainerProps<
  NodeType,
  EdgeType,
  Attributes
>['settings'] = {
  // 允许在无效容器中渲染 (默认值: false)
  allowInvalidContainer: true,
  defaultNodeType: 'circle',
  // 设置边的默认类型 (默认值: 'line')
  defaultEdgeType: 'arrow',

  // 相机缩放比例限制 (默认值: min: 0.1, max: 32)
  minCameraRatio: 0.2,
  maxCameraRatio: 5,

  // 节点渲染程序类 (默认值: {})
  nodeProgramClasses: {},

  // 渲染性能优化
  hideEdgesOnMove: true, // 移动时隐藏边 (默认值: false)
  hideLabelsOnMove: false, // 移动时隐藏标签 (默认值: false)
  renderLabels: true, // 是否渲染标签 (默认值: true)
  renderEdgeLabels: false, // 是否渲染边标签 (默认值: false)

  // 标签渲染优化
  labelDensity: 0.1, // 标签密度 (默认值: 0.03)
  labelGridCellSize: 100, // 标签网格大小 (默认值: 100)
  labelRenderedSizeThreshold: 8, // 标签大小阈值 (默认值: 6)

  // 默认样式
  defaultNodeColor: '#1a73e8', // 节点颜色 (默认值: '#999')
  defaultEdgeColor: '#ccc', // 边颜色 (默认值: '#ccc')

  // 交互体验优化
  zoomToSizeRatioFunction: (x: number) => x // 缩放比例函数 (默认值: Math.pow)
};
