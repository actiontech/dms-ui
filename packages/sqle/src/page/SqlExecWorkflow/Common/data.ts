export const SOURCE_WORKFLOW_PATH_KEY = 'sourceWorkflowId';
