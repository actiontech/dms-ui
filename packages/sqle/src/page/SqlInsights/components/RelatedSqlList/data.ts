import { GetSqlPerformanceInsightsRelatedSQLFilterSourceEnum } from '@actiontech/shared/lib/api/sqle/service/SqlInsight/index.enum';
import { t } from '../../../../locale';

export const RelatedSqlFilterSourceOptions = [
  {
    label: t('sqlInsights.relatedSqlList.source.order'),
    value: GetSqlPerformanceInsightsRelatedSQLFilterSourceEnum.workflow
  },
  {
    label: t('sqlInsights.relatedSqlList.source.sqlManage'),
    value: GetSqlPerformanceInsightsRelatedSQLFilterSourceEnum.sql_manage
  },
  {
    label: t('sqlInsights.relatedSqlList.source.workbench'),
    value: GetSqlPerformanceInsightsRelatedSQLFilterSourceEnum.workbench
  }
];
