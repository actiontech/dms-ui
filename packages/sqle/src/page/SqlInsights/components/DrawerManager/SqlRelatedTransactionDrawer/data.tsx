import { TransactionInfoLockTypeEnum } from '@actiontech/shared/lib/api/sqle/service/common.enum';
import { t } from '../../../../../locale';

export const TransactionInfoLockTypeEnumDict = {
  [TransactionInfoLockTypeEnum.EXCLUSIVE]: t(
    'sqlInsights.relatedSqlList.sqlRelatedTransaction.transactionInfo.lockTypeDict.exclusive'
  ),
  [TransactionInfoLockTypeEnum.SHARED]: t(
    'sqlInsights.relatedSqlList.sqlRelatedTransaction.transactionInfo.lockTypeDict.shared'
  ),
  [TransactionInfoLockTypeEnum.INTENTION_SHARED]: t(
    'sqlInsights.relatedSqlList.sqlRelatedTransaction.transactionInfo.lockTypeDict.intentionShared'
  ),
  [TransactionInfoLockTypeEnum.INTENTION_EXCLUSIVE]: t(
    'sqlInsights.relatedSqlList.sqlRelatedTransaction.transactionInfo.lockTypeDict.intentionExclusive'
  ),
  [TransactionInfoLockTypeEnum.SHARED_INTENTION_EXCLUSIVE]: t(
    'sqlInsights.relatedSqlList.sqlRelatedTransaction.transactionInfo.lockTypeDict.sharedIntentionExclusive'
  ),
  [TransactionInfoLockTypeEnum.ROW_LOCK]: t(
    'sqlInsights.relatedSqlList.sqlRelatedTransaction.transactionInfo.lockTypeDict.rowLock'
  ),
  [TransactionInfoLockTypeEnum.TABLE_LOCK]: t(
    'sqlInsights.relatedSqlList.sqlRelatedTransaction.transactionInfo.lockTypeDict.tableLock'
  ),
  [TransactionInfoLockTypeEnum.METADATA_LOCK]: t(
    'sqlInsights.relatedSqlList.sqlRelatedTransaction.transactionInfo.lockTypeDict.metadataLock'
  )
};

export const TransactionInfoLockTypeEnumFlagDict = {
  [TransactionInfoLockTypeEnum.SHARED]: 'S',
  [TransactionInfoLockTypeEnum.INTENTION_SHARED]: 'IS',
  [TransactionInfoLockTypeEnum.INTENTION_EXCLUSIVE]: 'IX',
  [TransactionInfoLockTypeEnum.SHARED_INTENTION_EXCLUSIVE]: 'SIX',
  [TransactionInfoLockTypeEnum.ROW_LOCK]: 'RL',
  [TransactionInfoLockTypeEnum.TABLE_LOCK]: 'TL',
  [TransactionInfoLockTypeEnum.METADATA_LOCK]: 'ML',
  [TransactionInfoLockTypeEnum.EXCLUSIVE]: 'X'
};
