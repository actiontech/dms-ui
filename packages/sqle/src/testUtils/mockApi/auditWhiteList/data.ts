import { IAuditWhitelistResV1 } from '@actiontech/shared/lib/api/sqle/service/common';
import { CreateAuditWhitelistReqV1MatchTypeEnum } from '@actiontech/shared/lib/api/sqle/service/common.enum';

export const auditWhiteListMockData: IAuditWhitelistResV1[] = [
  {
    audit_whitelist_id: 1,
    desc: 'test1',
    match_type: CreateAuditWhitelistReqV1MatchTypeEnum.exact_match,
    value: 'SELECT 1;',
    matched_count: 1,
    last_match_time: '2024-08-22T11:04:29.543+08:00'
  },
  {
    audit_whitelist_id: 2,
    desc: 'test2',
    match_type: CreateAuditWhitelistReqV1MatchTypeEnum.fp_match,
    value: 'SELECT 1;',
    matched_count: 10,
    last_match_time: undefined
  },
  {
    audit_whitelist_id: 3,
    desc: 'test3',
    match_type: CreateAuditWhitelistReqV1MatchTypeEnum.exact_match,
    value: 'SELECT 1;',
    matched_count: 0
  },
  {
    audit_whitelist_id: 4,
    desc: 'test4'
  }
];
