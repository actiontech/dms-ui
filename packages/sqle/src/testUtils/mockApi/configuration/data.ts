/**
 *  Generated by actionsky-cli api-mocks, do not modify this file unless necessary.
 *  When modifying the file, please also update the corresponding unit test files.
 */

import { ISSHPublicKeyInfo } from '@actiontech/shared/lib/api/sqle/service/common.d';

export const deriversMockData = [
  {
    driver_name: 'mysql',
    default_port: 3000,
    logo_url: 'test/logo?instance_type=mysql'
  }
];

export const testGitConnectionMockData = {
  is_connected_success: true,
  error_message: null,
  branches: ['main', 'test']
};

export const getSSHPublicKeyMockData = {
  public_key:
    'ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDzBOtcopZ1IdU51augExmKzRRYsJdNDnxVf5mlCRch2hMq4y02F4q4T71WFoLg22FptnxmY+LkOPgYyIvl6MdfBS6RQarc3aC2rO5bCbzuMf2f6+2hbYH3ACXJWAn4xX8b0ZK25t1u8jrXmAJPVXIMLEcLL4Wo7IqCQ1PwrkIeXuMfznHzVoAEv5o+jbUkj++I46AEmqrmDsgWCt83JK7ctBzhEXJWhF5rSyqGxXBptVPVMEO4ShJi2rKdk+ZbBGIAGu0db72hlvdqzUeU2Ng+J0dcIDsS/4pmEtQ+6uJFgslqF2Vt708tcOPTG4qdfFaK453PyceZ/F/f/BN+bPDJCh+oPQ3zdsKHmh0rXefICPMWe73QRLAyW/JeWcsnFlgoBmlwB4++Isc6xYx0dHFM/ws10vTJDif3CAi0VE1vKdSZvVxTFq5bZKhTX3zfA24yZzAl9BCOrQzWh20NWJ3Lgwk2JWL3NVsChKNk4aTCHNz1OyW/nojrd5tCyyqF06OqG1/xc5RmRkCH9gXqFA0Ngf3IdxJdpL9q3G60CBAi7Gfs6+gLIlMc6sgnlZdnyCj3Gmh7yRrmMZUDAmZnyvrYw3+4qQcVBi5gw64wuya2K6Hz6lrgxT9AlcUTNdHn1W6TimZtO3JUDclMUbA6euN/NOFsY4VpYPVNoMND3Lwesw=='
} as ISSHPublicKeyInfo;
