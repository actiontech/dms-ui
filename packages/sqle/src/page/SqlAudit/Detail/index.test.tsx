import SqlAuditDetail from '.';
import sqlAuditRecord from '../../../testUtils/mockApi/sqlAuditRecord';
import task from '../../../testUtils/mockApi/task';
import { BrowserRouter } from 'react-router-dom';
import { act, cleanup, screen } from '@testing-library/react';
import { renderWithThemeAndRedux } from '../../../testUtils/customRender';
import { mockUseCurrentUser } from '@actiontech/shared/lib/testUtil/mockHook/mockUseCurrentUser';
import { mockUseCurrentProject } from '@actiontech/shared/lib/testUtil/mockHook/mockUseCurrentProject';
import { mockProjectInfo } from '@actiontech/shared/lib/testUtil/mockHook/data';

describe('sqle/SqlAudit/Detail', () => {
  let mockUseCurrentProjectSpy: jest.SpyInstance;
  beforeEach(() => {
    jest.useFakeTimers();
    mockUseCurrentUser();
    mockUseCurrentProjectSpy = mockUseCurrentProject();
  });

  beforeAll(() => {
    jest.useRealTimers();
    cleanup();
  });

  it('should match snap shot', async () => {
    const getSqlAuditRecordDetailSpy = sqlAuditRecord.getSQLAuditRecord();
    const getAuditTaskSqlSpy = task.getAuditTaskSQLs();
    const { baseElement } = renderWithThemeAndRedux(
      <BrowserRouter>
        <SqlAuditDetail />
      </BrowserRouter>
    );
    await act(async () => jest.advanceTimersByTime(3000));
    expect(getSqlAuditRecordDetailSpy).toBeCalledTimes(1);
    expect(getAuditTaskSqlSpy).toBeCalledTimes(1);
    await act(async () => jest.advanceTimersByTime(3000));
    expect(baseElement).toMatchSnapshot();
  });

  it('should hide create button when project is archived', async () => {
    sqlAuditRecord.getSQLAuditRecord();
    mockUseCurrentProjectSpy.mockClear();
    mockUseCurrentProjectSpy.mockImplementation(() => ({
      ...mockProjectInfo,
      projectArchive: true
    }));
    renderWithThemeAndRedux(
      <BrowserRouter>
        <SqlAuditDetail />
      </BrowserRouter>
    );
    await act(async () => jest.advanceTimersByTime(3000));
    expect(screen.queryByText('创建审核')).not.toBeInTheDocument();
  });
});
