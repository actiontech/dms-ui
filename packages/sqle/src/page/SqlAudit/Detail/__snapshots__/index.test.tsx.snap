// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sqle/SqlAudit/Detail should match snap shot 1`] = `
<body>
  <div>
    <div
      class="ant-spin-nested-loading css-dev-only-do-not-override-txh9fw"
    >
      <div
        class="ant-spin-container"
      >
        <div
          class="actiontech-page-header-namespace fixed-style css-1xspr0w"
        >
          <div
            class="title"
          >
            <a
              href="/sqle/project/700300/sqlAudit"
            >
              <button
                class="ant-btn css-dev-only-do-not-override-674gwq ant-btn-default basic-button-wrapper css-geipcv"
                type="button"
              >
                <span
                  class="ant-btn-icon"
                >
                  <svg
                    fill="none"
                    height="16"
                    viewBox="0 0 16 16"
                    width="16"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M5.21863 7.33333H13.3333V8.66667H5.21863L8.79463 12.2427L7.85196 13.1853L2.66663 8L7.85196 2.81467L8.79463 3.75733L5.21863 7.33333Z"
                      fill="currentColor"
                    />
                  </svg>
                </span>
                <span>
                  返回SQL审核列表
                </span>
              </button>
            </a>
          </div>
          <div
            class="extra"
          >
            <a
              href="/sqle/project/700300/sqlAudit/create"
            >
              <button
                class="ant-btn css-dev-only-do-not-override-674gwq ant-btn-primary basic-button-wrapper css-geipcv"
                type="button"
              >
                <span
                  class="ant-btn-icon"
                >
                  <span
                    class="custom-icon custom-icon-add css-u3jcp4"
                    role="img"
                  >
                    <svg
                      fill="currentColor"
                      height="14"
                      viewBox="0 0 14 14"
                      width="14"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M6.41663 6.41663V2.91663H7.58329V6.41663H11.0833V7.58329H7.58329V11.0833H6.41663V7.58329H2.91663V6.41663H6.41663Z"
                      />
                    </svg>
                  </span>
                </span>
                <span>
                  创建审核
                </span>
              </button>
            </a>
          </div>
        </div>
        <div
          class="hasTopHeader clearPaddingBottom"
          style="height: 60px;"
        />
        <div
          class="css-17a4b1i"
        >
          <h3
            class="id-text"
          >
            审核ID
            : 
            123456
          </h3>
          <div
            class="tags-cont"
          >
            <div
              class="custom-tag-item"
            >
              审核状态
              ：
              <span
                class="ant-tag ant-tag-green basic-tag-wrapper basic-small-tag-wrapper css-ce1x2t css-dev-only-do-not-override-txh9fw"
              >
                审核成功
              </span>
            </div>
            <div
              class="custom-tag-item"
            >
              业务标签
              ：
              <span
                class="ant-tag ant-tag-geekblue basic-tag-wrapper basic-small-tag-wrapper css-n94hhh css-dev-only-do-not-override-txh9fw"
              >
                test
              </span>
            </div>
          </div>
        </div>
        <section
          class="css-zgknaj"
        >
          <div
            class="ant-row ant-row-space-between css-156vlto css-dev-only-do-not-override-txh9fw"
          >
            <div />
            <div
              class="ant-space css-dev-only-do-not-override-txh9fw ant-space-horizontal ant-space-align-center"
            >
              <div
                class="ant-space-item"
                style="margin-right: 4px;"
              >
                <div
                  class="css-rf9vxh"
                >
                  数据去重
                </div>
              </div>
              <div
                class="ant-space-item"
                style="margin-right: 4px;"
              >
                <div
                  class="ant-divider css-dev-only-do-not-override-txh9fw ant-divider-vertical"
                  role="separator"
                  style="height: 28px;"
                />
              </div>
              <div
                class="ant-space-item"
              >
                <button
                  class="ant-btn css-dev-only-do-not-override-674gwq ant-btn-default ant-btn-sm ant-btn-two-chinese-chars basic-button-wrapper css-geipcv"
                  type="button"
                >
                  <div
                    class="ant-space css-dev-only-do-not-override-674gwq ant-space-horizontal ant-space-align-center"
                  >
                    <div
                      class="ant-space-item"
                      style="margin-right: 5px;"
                    >
                      <span
                        class="custom-icon custom-icon-download css-u3jcp4"
                        role="img"
                      >
                        <svg
                          fill="currentColor"
                          height="14"
                          viewBox="0 0 14 14"
                          width="14"
                          xmlns="http://www.w3.org/2000/svg"
                        >
                          <path
                            d="M1.75 11.0833H12.25V12.25H1.75V11.0833ZM7.58333 7.68363L11.1247 4.14163L11.9496 4.96646L7 9.91663L2.05042 4.96704L2.87525 4.14163L6.41667 7.68246V1.16663H7.58333V7.68363Z"
                          />
                        </svg>
                      </span>
                    </div>
                    <div
                      class="ant-space-item"
                      style="margin-right: 5px;"
                    >
                      下载
                    </div>
                    <div
                      class="ant-space-item"
                    >
                      <span
                        class="custom-icon custom-icon-arrow-down css-u3jcp4"
                        role="img"
                      >
                        <svg
                          fill="currentColor"
                          height="14"
                          viewBox="0 0 14 14"
                          width="14"
                          xmlns="http://www.w3.org/2000/svg"
                        >
                          <path
                            d="M6.99999 7.68364L9.88749 4.79614L10.7123 5.62098L6.99999 9.33331L3.28766 5.62098L4.11249 4.79614L6.99999 7.68364Z"
                          />
                        </svg>
                      </span>
                    </div>
                  </div>
                </button>
              </div>
            </div>
          </div>
          <div
            class=" css-1x9vax"
          >
            <div
              class="ant-space css-dev-only-do-not-override-txh9fw ant-space-horizontal ant-space-align-center audit-result-filter-options"
            >
              <div
                class="ant-space-item"
                style="margin-right: 8px;"
              >
                <div
                  class="css-cl7oyl"
                >
                  <span
                    class="text"
                  >
                    全部等级
                  </span>
                </div>
              </div>
              <div
                class="ant-space-item"
                style="margin-right: 8px;"
              >
                <div
                  class="css-1bvs5o7"
                >
                  <span
                    class="text"
                  >
                    普通(Normal)
                  </span>
                </div>
              </div>
              <div
                class="ant-space-item"
                style="margin-right: 8px;"
              >
                <div
                  class="css-1bvs5o7"
                >
                  <span
                    class="text"
                  >
                    提示(Notice)
                  </span>
                </div>
              </div>
              <div
                class="ant-space-item"
                style="margin-right: 8px;"
              >
                <div
                  class="css-1bvs5o7"
                >
                  <span
                    class="text"
                  >
                    告警(Warning)
                  </span>
                </div>
              </div>
              <div
                class="ant-space-item"
              >
                <div
                  class="css-1bvs5o7"
                >
                  <span
                    class="text"
                  >
                    错误(Error)
                  </span>
                </div>
              </div>
            </div>
            <div
              class="ant-space css-dev-only-do-not-override-txh9fw ant-space-horizontal ant-space-align-center audit-result-info"
            >
              <div
                class="ant-space-item"
                style="margin-right: 20px;"
              >
                <div
                  class="audit-result-info-item"
                >
                  <span
                    class="audit-result-info-item-value audit-level-normal"
                  >
                    100
                    %
                  </span>
                  <span
                    class="audit-result-info-item-text"
                  >
                    审核通过率
                  </span>
                </div>
              </div>
              <div
                class="ant-space-item"
                style="margin-right: 20px;"
              >
                <div
                  class="audit-result-info-item"
                >
                  <span
                    class="audit-result-info-item-value audit-level-normal"
                  >
                    100
                  </span>
                  <span
                    class="audit-result-info-item-text"
                  >
                    审核结果评分
                  </span>
                </div>
              </div>
              <div
                class="ant-space-item"
              />
            </div>
          </div>
          <div
            class="ant-table-wrapper actiontech-table-namespace css-555x9n css-dev-only-do-not-override-bj9uhl"
          >
            <div
              class="ant-spin-nested-loading css-dev-only-do-not-override-bj9uhl"
            >
              <div
                class="ant-spin-container"
              >
                <div
                  class="ant-table ant-table-fixed-column ant-table-scroll-horizontal ant-table-has-fix-right"
                >
                  <div
                    class="ant-table-container"
                  >
                    <div
                      class="ant-table-content"
                      style="overflow-x: auto; overflow-y: hidden;"
                    >
                      <table
                        style="min-width: 100%; table-layout: auto;"
                      >
                        <colgroup>
                          <col
                            style="width: 100px;"
                          />
                          <col />
                          <col />
                          <col />
                          <col
                            style="width: 82px;"
                          />
                        </colgroup>
                        <thead
                          class="ant-table-thead"
                        >
                          <tr>
                            <th
                              class="ant-table-cell"
                              scope="col"
                            >
                              序号
                            </th>
                            <th
                              class="ant-table-cell audit-result-exec-sql-column"
                              scope="col"
                            >
                              执行语句
                            </th>
                            <th
                              class="ant-table-cell audit-result-column"
                              scope="col"
                            >
                              审核结果
                            </th>
                            <th
                              class="ant-table-cell audit-result-describe-column"
                              scope="col"
                            >
                              说明
                            </th>
                            <th
                              class="ant-table-cell actiontech-table-actions-column ant-table-cell-fix-right ant-table-cell-fix-right-first"
                              scope="col"
                              style="position: sticky; right: 0px;"
                            >
                              操作
                            </th>
                          </tr>
                        </thead>
                        <tbody
                          class="ant-table-tbody"
                        >
                          <tr
                            aria-hidden="true"
                            class="ant-table-measure-row"
                            style="height: 0px; font-size: 0px;"
                          >
                            <td
                              style="padding: 0px; border: 0px; height: 0px;"
                            >
                              <div
                                style="height: 0px; overflow: hidden;"
                              >
                                 
                              </div>
                            </td>
                            <td
                              style="padding: 0px; border: 0px; height: 0px;"
                            >
                              <div
                                style="height: 0px; overflow: hidden;"
                              >
                                 
                              </div>
                            </td>
                            <td
                              style="padding: 0px; border: 0px; height: 0px;"
                            >
                              <div
                                style="height: 0px; overflow: hidden;"
                              >
                                 
                              </div>
                            </td>
                            <td
                              style="padding: 0px; border: 0px; height: 0px;"
                            >
                              <div
                                style="height: 0px; overflow: hidden;"
                              >
                                 
                              </div>
                            </td>
                            <td
                              style="padding: 0px; border: 0px; height: 0px;"
                            >
                              <div
                                style="height: 0px; overflow: hidden;"
                              >
                                 
                              </div>
                            </td>
                          </tr>
                          <tr
                            class="ant-table-row ant-table-row-level-0"
                            data-row-key="1"
                          >
                            <td
                              class="ant-table-cell"
                            >
                              1
                            </td>
                            <td
                              class="ant-table-cell audit-result-exec-sql-column"
                            >
                              <div
                                class="css-4mzo37"
                              >
                                <div
                                  class="ant-typography ant-typography-ellipsis ant-typography-single-line ant-typography-ellipsis-single-line margin-bottom-0 css-dev-only-do-not-override-bj9uhl"
                                >
                                  <span>
                                    <span
                                      class="hljs-keyword"
                                    >
                                      SELECT
                                    </span>
                                     
                                    <span
                                      class="hljs-number"
                                    >
                                      1
                                    </span>
                                    ;
                                  </span>
                                </div>
                                <div
                                  class="css-1s8kb8e"
                                >
                                  <span
                                    aria-label="copy"
                                    class="anticon anticon-copy"
                                    role="img"
                                  >
                                    <svg
                                      aria-hidden="true"
                                      data-icon="copy"
                                      fill="currentColor"
                                      focusable="false"
                                      height="1em"
                                      viewBox="64 64 896 896"
                                      width="1em"
                                    >
                                      <path
                                        d="M832 64H296c-4.4 0-8 3.6-8 8v56c0 4.4 3.6 8 8 8h496v688c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8V96c0-17.7-14.3-32-32-32zM704 192H192c-17.7 0-32 14.3-32 32v530.7c0 8.5 3.4 16.6 9.4 22.6l173.3 173.3c2.2 2.2 4.7 4 7.4 5.5v1.9h4.2c3.5 1.3 7.2 2 11 2H704c17.7 0 32-14.3 32-32V224c0-17.7-14.3-32-32-32zM350 856.2L263.9 770H350v86.2zM664 888H414V746c0-22.1-17.9-40-40-40H232V264h432v624z"
                                      />
                                    </svg>
                                  </span>
                                </div>
                              </div>
                            </td>
                            <td
                              class="ant-table-cell audit-result-column"
                            >
                              <div>
                                <div
                                  class=" css-30pox0"
                                >
                                  <div
                                    class="css-1avyf41"
                                  >
                                    <span
                                      class="icon-wrapper"
                                    >
                                      <svg
                                        fill="none"
                                        height="20"
                                        viewBox="0 0 20 20"
                                        width="20"
                                        xmlns="http://www.w3.org/2000/svg"
                                      >
                                        <path
                                          d="M10.0001 18.3334C5.39758 18.3334 1.66675 14.6026 1.66675 10.0001C1.66675 5.39758 5.39758 1.66675 10.0001 1.66675C14.6026 1.66675 18.3334 5.39758 18.3334 10.0001C18.3334 14.6026 14.6026 18.3334 10.0001 18.3334ZM9.16925 13.3334L15.0609 7.44092L13.8826 6.26258L9.16925 10.9767L6.81175 8.61925L5.63341 9.79758L9.16925 13.3334Z"
                                          fill="#41BF9A"
                                        />
                                      </svg>
                                    </span>
                                    <span
                                      class="text-wrapper"
                                    >
                                      test message
                                    </span>
                                  </div>
                                </div>
                              </div>
                            </td>
                            <td
                              class="ant-table-cell audit-result-describe-column"
                            >
                              <div
                                aria-label="test"
                                class="ant-typography ant-typography-ellipsis ant-typography-single-line css-1am9q1l css-dev-only-do-not-override-bj9uhl"
                              >
                                test
                                <div
                                  aria-label="Edit"
                                  class="ant-typography-edit"
                                  role="button"
                                  style="border: 0px; background: transparent; padding: 0px; line-height: inherit; display: inline-block;"
                                  tabindex="0"
                                >
                                  <span
                                    class="custom-icon custom-icon-edit css-u3jcp4"
                                    role="img"
                                  >
                                    <svg
                                      fill="currentColor"
                                      height="16"
                                      viewBox="0 0 16 16"
                                      width="16"
                                      xmlns="http://www.w3.org/2000/svg"
                                    >
                                      <path
                                        d="M6.162 12.6667H14V14H2V11.1713L8.6 4.57133L11.428 7.40066L6.16133 12.6667H6.162ZM9.542 3.62933L10.9567 2.21466C11.0817 2.08968 11.2512 2.01947 11.428 2.01947C11.6048 2.01947 11.7743 2.08968 11.8993 2.21466L13.7853 4.10066C13.9103 4.22568 13.9805 4.39522 13.9805 4.57199C13.9805 4.74877 13.9103 4.91831 13.7853 5.04333L12.3707 6.45733L9.54267 3.62933H9.542Z"
                                      />
                                    </svg>
                                  </span>
                                </div>
                                <span
                                  aria-hidden="true"
                                  style="position: fixed; display: block; left: 0px; top: 0px; z-index: -9999; visibility: hidden; pointer-events: none; font-size: 0px; word-break: keep-all; white-space: nowrap;"
                                >
                                  lg
                                </span>
                                <span
                                  aria-hidden="true"
                                  style="position: fixed; display: block; left: 0px; top: 0px; z-index: -9999; visibility: hidden; pointer-events: none; font-size: 0px; width: 0px; white-space: normal; margin: 0px; padding: 0px;"
                                >
                                  <span
                                    aria-hidden="true"
                                  >
                                    ...
                                  </span>
                                  <div
                                    aria-label="Edit"
                                    class="ant-typography-edit"
                                    role="button"
                                    style="border: 0px; background: transparent; padding: 0px; line-height: inherit; display: inline-block;"
                                    tabindex="0"
                                  >
                                    <span
                                      class="custom-icon custom-icon-edit css-u3jcp4"
                                      role="img"
                                    >
                                      <svg
                                        fill="currentColor"
                                        height="16"
                                        viewBox="0 0 16 16"
                                        width="16"
                                        xmlns="http://www.w3.org/2000/svg"
                                      >
                                        <path
                                          d="M6.162 12.6667H14V14H2V11.1713L8.6 4.57133L11.428 7.40066L6.16133 12.6667H6.162ZM9.542 3.62933L10.9567 2.21466C11.0817 2.08968 11.2512 2.01947 11.428 2.01947C11.6048 2.01947 11.7743 2.08968 11.8993 2.21466L13.7853 4.10066C13.9103 4.22568 13.9805 4.39522 13.9805 4.57199C13.9805 4.74877 13.9103 4.91831 13.7853 5.04333L12.3707 6.45733L9.54267 3.62933H9.542Z"
                                        />
                                      </svg>
                                    </span>
                                  </div>
                                </span>
                              </div>
                            </td>
                            <td
                              class="ant-table-cell actiontech-table-actions-column ant-table-cell-fix-right ant-table-cell-fix-right-first"
                              style="position: sticky; right: 0px;"
                            >
                              <div
                                class="css-new95e"
                              >
                                <div
                                  class="ant-space css-dev-only-do-not-override-bj9uhl ant-space-horizontal ant-space-align-center"
                                >
                                  <div
                                    class="ant-space-item"
                                  >
                                    <button
                                      class="ant-btn css-dev-only-do-not-override-9n8g17 ant-btn-default ant-btn-sm actiontech-table-actions-button basic-button-wrapper css-geipcv"
                                      type="button"
                                    >
                                      <span>
                                        分 析
                                      </span>
                                    </button>
                                  </div>
                                </div>
                              </div>
                            </td>
                          </tr>
                        </tbody>
                      </table>
                    </div>
                  </div>
                </div>
                <ul
                  class="ant-pagination ant-table-pagination ant-table-pagination-right actiontech-table-pagination actiontech-table-pagination-fixed css-dev-only-do-not-override-bj9uhl"
                >
                  <li
                    class="ant-pagination-total-text"
                  >
                    <span>
                      共 1 条数据
                    </span>
                  </li>
                  <li
                    aria-disabled="true"
                    class="ant-pagination-prev ant-pagination-disabled"
                    title="Previous Page"
                  >
                    <button
                      class="ant-pagination-item-link"
                      disabled=""
                      tabindex="-1"
                      type="button"
                    >
                      <span
                        aria-label="left"
                        class="anticon anticon-left"
                        role="img"
                      >
                        <svg
                          aria-hidden="true"
                          data-icon="left"
                          fill="currentColor"
                          focusable="false"
                          height="1em"
                          viewBox="64 64 896 896"
                          width="1em"
                        >
                          <path
                            d="M724 218.3V141c0-6.7-7.7-10.4-12.9-6.3L260.3 486.8a31.86 31.86 0 000 50.3l450.8 352.1c5.3 4.1 12.9.4 12.9-6.3v-77.3c0-4.9-2.3-9.6-6.1-12.6l-360-281 360-281.1c3.8-3 6.1-7.7 6.1-12.6z"
                          />
                        </svg>
                      </span>
                    </button>
                  </li>
                  <li
                    class="ant-pagination-item ant-pagination-item-1 ant-pagination-item-active"
                    tabindex="0"
                    title="1"
                  >
                    <a
                      rel="nofollow"
                    >
                      1
                    </a>
                  </li>
                  <li
                    aria-disabled="true"
                    class="ant-pagination-next ant-pagination-disabled"
                    title="Next Page"
                  >
                    <button
                      class="ant-pagination-item-link"
                      disabled=""
                      tabindex="-1"
                      type="button"
                    >
                      <span
                        aria-label="right"
                        class="anticon anticon-right"
                        role="img"
                      >
                        <svg
                          aria-hidden="true"
                          data-icon="right"
                          fill="currentColor"
                          focusable="false"
                          height="1em"
                          viewBox="64 64 896 896"
                          width="1em"
                        >
                          <path
                            d="M765.7 486.8L314.9 134.7A7.97 7.97 0 00302 141v77.3c0 4.9 2.3 9.6 6.1 12.6l360 281.1-360 281.1c-3.9 3-6.1 7.7-6.1 12.6V883c0 6.7 7.7 10.4 12.9 6.3l450.8-352.1a31.96 31.96 0 000-50.4z"
                          />
                        </svg>
                      </span>
                    </button>
                  </li>
                  <li
                    class="ant-pagination-options"
                  >
                    <div
                      aria-label="Page Size"
                      class="ant-select ant-pagination-options-size-changer css-dev-only-do-not-override-bj9uhl ant-select-single ant-select-show-arrow ant-select-show-search"
                    >
                      <div
                        class="ant-select-selector"
                      >
                        <span
                          class="ant-select-selection-search"
                        >
                          <input
                            aria-activedescendant="rc_select_TEST_OR_SSR_list_0"
                            aria-autocomplete="list"
                            aria-controls="rc_select_TEST_OR_SSR_list"
                            aria-expanded="false"
                            aria-haspopup="listbox"
                            aria-label="Page Size"
                            aria-owns="rc_select_TEST_OR_SSR_list"
                            autocomplete="off"
                            class="ant-select-selection-search-input"
                            id="rc_select_TEST_OR_SSR"
                            role="combobox"
                            type="search"
                            value=""
                          />
                        </span>
                        <span
                          class="ant-select-selection-item"
                          title="20 / page"
                        >
                          20 / page
                        </span>
                      </div>
                      <span
                        aria-hidden="true"
                        class="ant-select-arrow"
                        style="user-select: none;"
                        unselectable="on"
                      >
                        <span
                          aria-label="down"
                          class="anticon anticon-down ant-select-suffix"
                          role="img"
                        >
                          <svg
                            aria-hidden="true"
                            data-icon="down"
                            fill="currentColor"
                            focusable="false"
                            height="1em"
                            viewBox="64 64 896 896"
                            width="1em"
                          >
                            <path
                              d="M884 256h-75c-5.1 0-9.9 2.5-12.9 6.6L512 654.2 227.9 262.6c-3-4.1-7.8-6.6-12.9-6.6h-75c-6.5 0-10.3 7.4-6.5 12.7l352.6 486.1c12.8 17.6 39 17.6 51.7 0l352.6-486.1c3.9-5.3.1-12.7-6.4-12.7z"
                            />
                          </svg>
                        </span>
                      </span>
                    </div>
                  </li>
                </ul>
              </div>
            </div>
          </div>
        </section>
      </div>
    </div>
  </div>
</body>
`;
